build-elf32.sh

```diff
@@ -219,15 +219,14 @@ else
     pch_opt=
 fi
 
-# GCC + configure of libstdc++
+# GCC must be built in 2 stages. First minimal GCC build is for building
+# newlib and second stage is a complete GCC with newlib headers. See:
+# http://www.ifp.illinois.edu/~nakazato/tips/xgcc.html
 build_dir_init gcc
-configure_elf32 gcc gcc --with-newlib $pch_opt
-make_target building all-gcc all-target-libgcc
-make_target installing ${HOST_INSTALL}-gcc install-target-libgcc
-if [ "$DO_PDF" = "--pdf" ]
-then
-    make_target "generating PDF documentation" install-pdf-gcc
-fi
+configure_elf32 gcc gcc --without-headers --with-newlib $pch_opt
```

Comment:
First stage cn be configured without pch_out. - this is only for second stage, where libstdc++ will be actually built.